README.md

```diff
@@ -14,6 +14,8 @@ A design system and Vue.js component library created by [Schwarz IT](https://it.
 
 <br />
 
-## [Documentation](https://onyx.schwarz)
+## Documentation
+
+You can find our documentation [here](https://onyx.schwarz).
 
 ## [Contributing](CONTRIBUTING.md)
```

Comment:
do you want to split the link/headline for the contributing part as well? :) like with # Documentation